lib/vsc/utils/exceptions.py

```diff
@@ -81,6 +84,27 @@ def __init__(self, msg, *args, **kwargs):
         if args:
             msg = msg % args
 
+        if self.LOC_INFO_TOP_PKG_NAMES is not None:
+            # determine correct frame to fetch location information from
+            frames_up = 1
+            if self.__class__ != LoggedException:
+                # move a level up when this instance is derived from LoggedException
+                frames_up += 1
+
+            # figure out where error was raised from
+            # current frame: this constructor, one frame above: location where this EasyBuildError was created/raised
+            frameinfo = inspect.getouterframes(inspect.currentframe())[frames_up]
+
+            # determine short location of Python module where error was raised from (starting with 'easybuild/' or 'vsc/')
+            path_parts = frameinfo[1].split(os.path.sep)
+            relpath = path_parts.pop()
+            while path_parts and not any([relpath.startswith(n + os.path.sep) for n in self.LOC_INFO_TOP_PKG_NAMES]):
```

Comment:
`relpath` can't have a `os.path.sep`, because it's an element of a `.split(os.path.sep)`, so the `any` is always false.
i have no idea what you are trying to do here tough